doc/rados/troubleshooting/log-and-debug.rst

```diff
@@ -2,177 +2,204 @@
+Subsystems that are common to all daemons are set under ``[global]`` in the
+configuration file. Subsystems for a specific daemon are set under the relevant
+daemon section in the configuration file (for example, ``[mon]``, ``[osd]``,
+``[mds]``). Here is an example that shows possible debugging settings in a Ceph
+configuration file:
 
 .. code-block:: ini
 
-	[global]
-		debug_ms = 1/5
-		
-	[mon]
-		debug_mon = 20
-		debug_paxos = 1/5
-		debug_auth = 2
-		 
- 	[osd]
- 		debug_osd = 1/5
- 		debug_filestore = 1/5
- 		debug_journal = 1
- 		debug_monc = 5/20
- 		
-	[mds]
-		debug_mds = 1
-		debug_mds_balancer = 1
+    [global]
+        debug_ms = 1/5
+        
+    [mon]
+        debug_mon = 20
+        debug_paxos = 1/5
+        debug_auth = 2
+         
+     [osd]
+         debug_osd = 1/5
+         debug_filestore = 1/5
+         debug_journal = 1
+         debug_monc = 5/20
+         
+    [mds]
+        debug_mds = 1
+        debug_mds_balancer = 1
 
 
-See `Subsystem, Log and Debug Settings`_ for details.
+For details, see `Subsystem, Log and Debug Settings`_.
 
 
 Accelerating Log Rotation
 =========================
 
-If your OS disk is relatively full, you can accelerate log rotation by modifying
-the Ceph log rotation file at ``/etc/logrotate.d/ceph``. Add  a size setting
-after the rotation frequency to accelerate log rotation (via cronjob) if your
-logs exceed the size setting. For example, the  default setting looks like
-this::
-   
-	rotate 7
-  	weekly
-  	compress
-  	sharedscripts
-   	
-Modify it by adding a ``size`` setting. ::
-   
-  	rotate 7
-  	weekly
-  	size 500M
-  	compress
-  	sharedscripts
-
-Then, start the crontab editor for your user space. ::
-   
-  	crontab -e
-	
-Finally, add an entry to check the ``etc/logrotate.d/ceph`` file. ::
-   
-  	30 * * * * /usr/sbin/logrotate /etc/logrotate.d/ceph >/dev/null 2>&1
-
-The preceding example checks the ``etc/logrotate.d/ceph`` file every 30 minutes.
+If your OS disk is nearly full, you can accelerate log rotation by modifying
+the Ceph log rotation file at ``/etc/logrotate.d/ceph``. To use cronjob to
+accelerate log rotation whenever logs exceed a specified size, add a size
+setting after the rotation-frequency setting. The procedure for adding the size
+setting can be illustrated as follows. 
+
+#. Note the default settings of the ``/etc/logrotate.d/ceph`` file::
+
+      rotate 7
+      weekly
+      compress
+      sharedscripts
+
+#. Modify them by adding a ``size`` setting::
+
+      rotate 7
+      weekly
+      size 500M
+      compress
+      sharedscripts
 
+#. Start the crontab editor for your user space:
+
+   .. prompt:: bash $
+
+      crontab -e
+
+#. Add an entry to crontab that instructs cron to check the
+   ``etc/logrotate.d/ceph`` file::
+
+      30 * * * * /usr/sbin/logrotate /etc/logrotate.d/ceph >/dev/null 2>&1
+
+In this example, the ``etc/logrotate.d/ceph`` file will be checked every 30
+minutes.
```

Comment:
I'm sorely tempted to suggest dropping a file in `/etc/cron.d` instead.

Reply:
Let's do that in a separate PR.